model_api/python/model_api/tilers/tiler.py

```diff
@@ -93,6 +93,10 @@ def parameters(cls):
                     max=1.0,
                     description="Overlap of tiles",
                 ),
+                "tile_with_full_img": BooleanValue(
+                    default_value=False,
```

Comment:
This flag should be true by default to preserve backward compatibility.

Reply:
Also, this'd fix the python accuracy test.

Reply:
Thanks for the comment, and updated.